src/Files.App/UserControls/StatusBarControl.xaml

```diff
@@ -163,26 +163,47 @@
 			Orientation="Horizontal"
 			Spacing="4">
 
-			<!--  Open in VS Code Button  -->
+			<!--  Open in IDE Button  -->
 			<Button
-				x:Name="OpenInVSCodeButton"
+				x:Name="OpenInIDEButton"
 				Height="24"
 				Padding="8,0,8,0"
 				VerticalAlignment="Center"
-				x:Load="{x:Bind converters:MultiBooleanConverter.AndNotConvert(Commands.OpenInVSCode.IsExecutable, Commands.OpenInVS.IsExecutable), Mode=OneWay}"
+				x:Load="{x:Bind Commands.OpenInVS.IsExecutable, Mode=OneWay}"
```

Comment:
In the future, we might want to make this button available for non git folders as well.

Reply:
It doesn't seem to work properly...
At the moment the button is visbile only if there's a `.sln` file in the current folder. If you explore subfolders the button will be hidden

Reply:
I fixed this in the most recent commit